Kitodo/src/main/resources/kitodo_config.properties

```diff
@@ -71,6 +71,9 @@ directory.modules=/usr/local/kitodo/modules/
 # Falls Dateien zum Debuggen / Tracen geschrieben werden sollen, hier ein Verzeichnis angeben
 directory.debug=/usr/local/kitodo/debug/
 
+# Absolute path to the directory that the OCR-D workflow files will be
+# read from. It must be terminated by a directory separator ("/").
+ocrd.workflows.directory=/usr/local/kitodo/ocrd/workflows/
```

Comment:
Are there going to be other OCRD files apart from workflows in the future? Otherwise I am a little confused about this nested folder structure - why not use just `ocrd_worfklows` or something like that?

Reply:
Currently, no additional files are planned. The path is adjustable, but I have now changed the default to your suggestion.